templates/next/components/common/Pagination.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from "next/link";
+import { FunctionComponent } from "react";
+import { PagedCollection } from "../../types/Collection";
+
+interface Props {
+  collection: PagedCollection<any>;
+}
+
+const Pagination: FunctionComponent<Props> = ({ collection }) => {
```

Comment:
Using `FunctionComponent` is not recommended.
See https://github.com/facebook/create-react-app/pull/8177.

Reply:
Should I just update the typehint as following?
```
const Pagination = ({ collection }: Props) => {
```

Reply:
Yes!